### PR TITLE
Add ERC: ERC-721 Guarantee Extension

### PR DESCRIPTION
**Pull request by iunknow588**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-798.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+eip: EIP-798
```
Comment (abcoathup) [inline ERCS/erc-798.md:2]:
```suggestion
eip: 7652
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: ERCS/erc-798.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+title: ERC-798, EIP-721 Guarantee Extension
```
Comment (abcoathup) [inline ERCS/erc-798.md:3]:
```suggestion
title: ERC-721 Guarantee Extension
```
Don't include ERC number in title

File: ERCS/erc-798.md (hunk 1 above)
Comment (iunknow588) [reply ERCS/erc-798.md]: Thanks a lot,it has been modified as suggested.

File: ERCS/erc-798.md (hunk 2 above)
Comment (iunknow588) [reply ERCS/erc-798.md]: Thanks a lot,it has been modified as suggested.

**Review by iunknow588 — COMMENTED** [review]
What should I do next?

**Pull request closed without merging**